src/sinks/websocket/sink.rs

```diff
@@ -261,6 +261,13 @@ impl WebSocketSink {
 
         let bytes_sent = register!(BytesSent::from(Protocol("websocket".into())));
         let events_sent = register!(EventsSent::from(Output(None)));
+        let encode_as_binary = match self.encoder.serializer() {
+            codecs::encoding::Serializer::RawMessage(_)
+            | codecs::encoding::Serializer::Avro(_)
+            | codecs::encoding::Serializer:: Native(_) => true,
+            _ => false,
```

Comment:
Sorry, I should have mentioned this nit before, but could we list out all of the codecs rather than having a default? My worry is that we'll add a new one that should be encoded as binary and forget to update this list.

Reply:
updated